app/jenkins-for-jira-ui/src/components/JenkinsSetup/JenkinsSetup.tsx

```diff
@@ -214,10 +261,17 @@ const JenkinsSetup = (): JSX.Element => {
 	const [serverName, setServerName] = useState('');
 	const [showMyJenkinsAdmin, setShowMyJenkinsAdmin] = useState(false);
 	const [showIAmTheJenkinsAdmin, setShowIAmTheJenkinsAdmin] = useState(false);
+	const [isCopyAdminSiteName, setIsCopyAdminSiteName] = useState(false);
```

Comment:
I'm not sure about the context of why you are doing this. Could you explain?

Either way, one piece of state could manage all of these if you did something like:

```
const [isCopyName, setIsCopyName] = useState({
  buttonOne: false,
  buttonTwo: false
});
```

Reply:
Basically each states correspond to whether the value is copied or not, which is used to decide if user can continue to next page.